examples/lighting-app/nrfconnect/main/AppTask.cpp

```diff
@@ -360,6 +363,8 @@ void AppTask::ActionInitiated(LightingManager::Action_t aAction)
     {
         LOG_INF("Turn Off Action has been initiated");
     }
+
+    sAppTask.mButtonTriggeredActon = (aActor == AppEvent::kEventType_Button) ? true : false;
```

Comment:
"Action" instead of "Acton"?  Also what's `aActor`?

But most importantly, do we want to store this in a global variable?  What happens if another action is initiated before the previous one completes?